### DISCUSSION TITLE
Matrix chatroom

### DISCUSSION THREAD

**Discussion by natrius** [nextcloud/recognize#1203] [General]
@marcelklehr mentioned in this https://github.com/nextcloud/recognize/discussions/656#discussioncomment-4854359 post a matrix-chatroom. But i could nowhere find a link to said room. 
Unfortunately i thought a bit to late to search in discussions because i already created a room. Or is the room mentioned on another main-server?

The one i created with a set up draupnir-bot to fight spam https://matrix.to/#/#recognize:matrix.org

It is intended for quick questions and chatter around recognize, that should not be in a discussion or github issue yet.

**Comment by marcelklehr** [comment]
Hey @natrius 

Thank you for opening this discussion. The room I meant is this one: https://matrix.to/#/#marcelklehr_recognize:gitter.im

It's hosted on gitter, but I'm happy to add your room to the README as well.
